src/Controller/Component/AuthComponent.php

```diff
@@ -384,13 +385,10 @@ protected function _unauthenticated(Controller $controller)
                 $this->_config['ajaxLogin'],
                 $this->RequestHandler->ajaxLayout
             );
-            $response->statusCode(403);
-
-            return $response;
+            return $response->withStatus(403);
```

Comment:
Do we even need that line if the next one returns the $response then anway like this?